composer.json

```diff
@@ -25,7 +25,8 @@
         "jgm/CommonMark": "0.18",
         "michelf/php-markdown": "~1.4",
         "phpunit/phpunit": "~4.3",
-        "phpunit/phpunit-mock-objects": "2.3.0"
+        "phpunit/phpunit-mock-objects": "2.3.0",
```

Comment:
Yeah, 2.3.1 uses a reflection feature that's broken in HHVM.  I've temporarily locked this version until HHVM can resolve the issue:  https://github.com/thephpleague/commonmark/issues/89